### PR TITLE
Componentize projects

### PR DESCRIPTION
**Pull request by dince16**
**What kind of change does this PR introduce?**
This pull request adds the following changes to the project

1. Creates a ProjectTile component and uses it in both the projects page and the home page.
2. Changes all images associated with projects to fluid which fixes the resizing issues. (This is across the home page, projects page and specific project pages.)
3. Changes all images associated with projects to PreviewCompatibleImages to match the about page. They are also formatted to be JPGs.

**Does this PR introduce a breaking change?**
At the time of this pull request I did notice a small bug. (I'm creating the pr anyway but the bug should be revisited.) When you open the projects page, the first description doesn't appear. It isn't until you hard refresh in the browser that you see the description. This is consistent across all routes a user could take to navigate to the projects page.

### CODE REVIEW

File: src/components/ProjectTile.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+const ProjectTile = ({ data }) => (
+  <div className="columns">
+      <div key={data.id} className="column">
+          <li style={{display:'inline-block', padding:'5%'}} key={data.id}>
+            <Link to={`/${data.id}`}><PreviewCompatibleImage imageInfo={data.project_image} /></Link>
+            <h3 className="has-text-weight-bold is-size-5">
+            <Link style={{display:'inline-block', color: '#1C2833'}} to={`/${data.id}`}>{data.project_name}</Link>
+            </h3>
+            <h4 className="has-text-weight-normal is-size-5 is-size-6" style={{color: '#1C2833'}}>
+            <Link style={{color: '#1C2833'}} to={`/${data.profiles.profile_name}`}><i>Sponsored by {data.profiles.profile_name}</i></Link>
```
[the remark this answers is not in the document]
Comment (dince16) [reply src/components/ProjectTile.js]: Yeah so this is something that was documented in the last pr. This field is a relationship to another table in the database and I haven't figured out how to make them talk perfectly yet. All profiles are filled out in the database but i'm still working to troubleshoot why they aren't showing up. My goal was to keep looking into it as we developed the projects page further. Is this ok or should I comment it out until there is full functionality?

[the remark this answers is not in the document]
Comment (chris-hammond) [reply src/components/ProjectTile.js]: I really want "done" to be done, *but* I already forked off of this branch... So let's merge it as-is and add a trello card to fix it.

**Comment by kjh61998** [comment]
One thing I noticed is that on every page, somehow "Success Stories" got added to the bottom of each page. I don't know when it happened because it doesn't seem to be in the files changed in this pull request.  Should we still merge this and then have someone remove it for the next pull request?

**Comment by drohwedderGFU** [comment]
@chris-hammond 
The issue that @kjh61998 brought up is still present, how are we going to address this?

**Pull request merged**